clients/da-vinci-client/src/main/java/com/linkedin/davinci/utils/ByteArrayKey.java

```diff
@@ -31,6 +31,10 @@ public boolean equals(Object o) {
     return Arrays.equals(content, that.content);
   }
 
+  public byte[] getContent() {
+    return this.content;
+  }
```

Comment:
This seems like a dangerous change... the `content` field is intended to be immutable, but there is no way to ensure this besides encapsulation and visibility rules, but we are now violating that with this getter... Since the goal is to use it in a comparator, can we instead make this class implement `Comparable`? It seems that it would achieve the same goal while preserving encapsulation.

Reply:
Fixed.